addons/binding/org.openhab.binding.silvercrestwifisocket/README.md

```diff
@@ -1,43 +1,45 @@
-# Silvercrest Wifi Plug Binding
-
-This binding integrates the Silvercrest Wifi Socket SWS-A1 sold by Lidl.
-
-## Supported Things
-
-- Silvercrest Wifi Socket SWS-A1 - [(Owner Manual)](http://www.lidl-service.com/static/118127777/103043_FI.pdf)   --   Tested with firmware version: 1.41, 1.60, 1.70
-
-
-## Discovery
-
-The Discovery of Wifi Sockets is always running in the background. If a command is sent to wifi socket using the android/ios app or if the physical button in the device is pressed, the device will be recognized and will be placed in the Inbox.
-
-## Binding Configuration
-
-The binding does not require any special configuration. The Wifi Socket should be connected to the same wifi network.
-
-## Thing Configuration
-
-To configure a Wifi Socket manually only the mac address is required. You can check the Wifi Socket mac address in your router or using some mobile app.
-
-Wifi Socket thing parameters:
-
-| Parameter ID | Parameter Type | Mandatory | Description | Default |
-|--------------|----------------|------|------------------|-----|
-| macAddress | text | true | The socket MAC address |  |
-| hostAddress | text | false | The socket Host address. The binding is capable to discover the host address. |  |
-| updateInterval | integer | false | Update time interval in seconds to request the status of the socket. | 60 |
-
-
-E.g.
-
-```
-Thing silvercrestwifisocket:wifiSocket:lamp [ macAddress="ACCF23343C50" ]
-```
-
-## Channels
-
-The Silvercrest Wifi Socket support the following channel:
-
-| Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|
+# Silvercrest Wifi Plug Binding
+
+This binding integrates the Silvercrest Wifi Socket SWS-A1 sold by Lidl and the EasyHome Wifi Socket DS-124 sold by Aldi (excluding 433 MHz control).
+
+## Supported Things
+
+- Silvercrest Wifi Socket SWS-A1 - [(Owner Manual)](http://www.lidl-service.com/static/118127777/103043_FI.pdf)   --   Tested with firmware version: 1.41, 1.60, 1.70
+- EasyHome Wifi Socket DIS-124 (https://www.aldi-sued.de/de/infos/aldi-sued-a-bis-z/s/serviceportal/ergebnisliste/sis/si/wifi-steckdose/)
+
+
+## Discovery
+
+The Discovery of Wifi Sockets is always running in the background. If a command is sent to wifi socket using the android/ios app or if the physical button in the device is pressed, the device will be recognized and will be placed in the Inbox.
+
+## Binding Configuration
+
+The binding does not require any special configuration. The Wifi Socket should be connected to the same wifi network.
+
+## Thing Configuration
+
+To configure a Wifi Socket manually the mac address and the vendor is required. You can check the Wifi Socket mac address in your router or using some mobile app. Supported vendors are either Silvercrest (Lidl) or EasyHome (Aldi). 
+
+Wifi Socket thing parameters:
+
+| Parameter ID | Parameter Type | Mandatory | Description | Default |

```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍

Reply:
Will do it...